modules/nw-rfhe-creating_bmc_event_sub.adoc

```diff
@@ -0,0 +1,164 @@
+// Module included in the following assemblies:
```

Comment:
this seem to be a duplicate of file modules/nw-rfhe-creating-bmc-event-sub.adoc with a different name.